src/loaders/iff.c

```diff
@@ -146,10 +146,8 @@ int libxmp_iff_load(iff_handle opaque, struct module_data *m, HIO_HANDLE *f, voi
 
 	while (!hio_eof(f)) {
 		ret = iff_chunk(opaque, m, f, parm);
-
```

Comment:
See previous comments.

Reply:
This version is more readable to me btw..

Reply:
kept this one.